global.json

```diff
@@ -3,7 +3,7 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "6.0.100-preview.4.21255.9",
+    "dotnet": "6.0.100-preview.6.21355.2",
```

Comment:
@Forgind You were doing this in the RAR-precache branch, right? Probably best to work through the issues in this PR rather than there.